### PR TITLE
Apply ivi10 baseline

### PR DESCRIPTION
**Pull request by chbae**
# Task description

meta-ivi 10.0.0 is using yocto jethro.
# Poky

URI:    git://git.yoctoproject.org/poky
branch: jethro
revision:   fc45deac89ef63ca1c44e763c38ced7dfd72cbe1
# meta-oe

URI:    git://git.openembedded.org/meta-openembedded
branch: jethro
revision:   ad6133a2e95f4b83b6b3ea413598e2cd5fb3fd90
# meta-ivi

URI: http://git.yoctoproject.org/git/meta-ivi
tag: 10.0.0
revision: 9d72380c1d50d0c3469b85c3a43fe612b5ee1dd9

See meta-ivi 10 wiki page: https://at.projects.genivi.org/wiki/pages/viewpage.action?pageId=11567697
Check the updates that come with the new poky version Jethro

[GDP-212] Upgrade GDP to yocto 2.0 (jethro)

### CODE REVIEW

**Comment by chbae** [comment]
@tom--pollard 
I updated two commits.
- On RPI2 and Minnnowbard: I successfully built and tested
- On qemux86-64: Successfully built I didn't test image yet because of my PC problem.
- On porter: Got build error (Need to update meta-renesas)

**Comment by tom--pollard** [comment]
@chbae thanks for testing, and yes we need meta-renesas to have a jethro compatible layer. What is your opinion on switching to jethro branch of qt5? I know this will bring more issues as it is beyond the 'core' layer upgrade

**Comment by chbae** [comment]
@tom--pollard 
IMO qt upgrade and this work need to be separated.

**Comment by tom--pollard** [comment]
I agree that it is separate work @chbae , I'm just unsure if GDP can declare itself yocto 2.0 before all layers are jethro. However, if GDP10=meta-ivi10 then that condition is met with this core layer PR.

**Comment by chbae** [comment]
@tom--pollard 
I thought that meta-ivi10 is mandatory and other features are option. yocto 2.0 is meta-ivi10's requirement.
However other features are also really important.

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager_7.0.bbappend [hunk 1]
```diff
@@ -2,12 +2,13 @@ FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
 
 DEPENDS_append = " pulseaudio"
 
-SRCREV = ""
```
Comment (tom--pollard) [inline meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager_7.0.bbappend:5]: Is there documentation on why this is required to be set in 2.0? We purposely tracked the tag here @chbae

Comment (chbae) [inline meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager_7.0.bbappend:5]: @tom--pollard 
If I set the empty value, I got the error. So I set this variable.

**Comment by gunnarx** [comment]
Ok for this one the pipeline seems to fail on a similar layer mismatch as the other PR if I see it correctly.

meta-ivi 10 has weston_1.9.0.bbappend but it can't find weston_1.9.0.bb anywhere.
The genivi-dev-platform it is tested against is checking out a version of poky that only has weston_1.6.0.bb

The PR checks performed correctly as far as I can tell.  It is just tricky with these multi-layer dependencies.

**Comment by chbae** [comment]
@gunnarx 
meta-ivi has webson_1.9.0.bb in meta-ivi/meta-ivi/recipes-graphics/wayland/weston_1.9.0.bb

@tom--pollard 
Please review this commit and then merge :)

**Comment by tom--pollard** [comment]
For anyone who isn't tracking this on JIRA I've been able to build and deploy for porter on a meta-renesas ivi10/yocto2.0 based feature branch.

@chbae We will also have to update the readme, do you think it is time to stop listing all the layers + specific shas?

**Comment by gunnarx** [comment]
> @gunnarx
> meta-ivi has webson_1.9.0.bb in meta-ivi/meta-ivi/recipes-graphics/wayland/weston_1.9.0.bb

Yes I know but that is in meta-ivi 10.0 branch right?  The GDP that Go was building the PR against seems to be GDP-9 / meta-ivi 9 based, so it therefore failed.  Just to confirm, do you agree that the PR build functioned correctly and this is a case of multiple-layer update problem?  @tom--pollard please comment.

**Comment by tom--pollard** [comment]
@gunnarx I've built this PR along with the related one in m-g-d for porter,qemu & minnowboard, all built successfully, will be the case of multiple / parallel PR's needing to be built together

**Comment by chbae** [comment]
This should be merged with https://github.com/GENIVI/genivi-dev-platform/pull/14.
So the go.cd ci can be failed. I'll merge it.

**Pull request merged**